File: .ProgramingBasics/ComplexConditionalStatements/11.Cinema/src/Cinema.java
import java.util.Scanner;

public class Cinema {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        int row = Integer.parseInt(scanner.nextLine());
        int col = Integer.parseInt(scanner.nextLine());
        double price = 0;

        switch (input) {
            case "Premiere":
                price = 12.00;
                System.out.printf("%.2f leva", price * row * col);
                break;
            case "Normal":
                price = 7.50;
                System.out.printf("%.2f leva", price * row * col);
                break;
            case "Discount":
                price = 5.00;
                System.out.printf("%.2f leva", price * row * col);
                break;
        }
    }
}

File: ProgramingBasics/ComplexConditionalStatements/11.Cinema/11.Cinema/Cinema.cs
using System;

namespace _11.Cinema
{
    public class Cinema
    {
        public static void Main(string[] args)
        {
            var input = Console.ReadLine();
            var row = int.Parse(Console.ReadLine());
            var col = int.Parse(Console.ReadLine());
            var price = 0d; ;

            if (input.Equals("Premiere"))
            {
                price = 12.00;
                Console.WriteLine("{0:F2} leva", price * row * col);
            }
            else if (input.Equals("Normal"))
            {
                price = 7.50;
                Console.WriteLine("{0:F2} leva", price * row * col);
            }
            else if (input.Equals("Discount"))
            {
                price = 5.00;
                Console.WriteLine("{0:F2} leva", price * row * col);
            }
        }
    }
}
